go-controller/pkg/ovn/egressqos.go

```diff
@@ -1021,3 +1048,40 @@ func (oc *DefaultNetworkController) syncEgressQoSNode(key string) error {
 
 	return nil
 }
+
+func (oc *DefaultNetworkController) setEgressQoSStatus(egressQoS *egressqosapi.EgressQoS, handlerErr error) error {
```

Comment:
is this a copy of what you have in ClusterManager? seems like time for utils...

Reply:
This is just updating `status.Messages` whereas method in the ClusterManager is for updating `status.status`.